content/cumulus-linux/Layer-2/LACP-Bypass.md

```diff
@@ -73,6 +73,12 @@ cumulus@switch:~$ net pending
 cumulus@switch:~$ net commit
 ```
 
+{{%notice note%}}
+
+Clag-ID is not a required parameter in the configuration shown above. While LACP-Bypass is often configured on Bonds involved in MLAG, MLAG is not required to use LACP-Bypass.
+
+{{%/notice%}}
+
```

Comment:
I'd get rid of some unneeded capitalization, so word it more like this:

`clag id` is not a required parameter in the configuration shown above. While LACP bypass is often configured on bonds involved in MLAG, MLAG is not required to use LACP bypass.